opds.py

```diff
@@ -506,6 +506,10 @@ def groups(cls, _db, title, url, lane, annotator,
             return cached
 
         feed_size = Configuration.featured_lane_size()
+        if ignore_feed_size:
+            # This groups feed should return groups no matter how many
+            # books are found.
+            feed_size = 1
```

Comment:
I don't understand how this solves the problem. It seems like you'd want to check this around line 541?